README.md

```diff
@@ -1342,6 +1342,12 @@ Ensure all eligible online accounts have [multi-factor authentication](https://e
 
 Encrypt files locally before backing them up to external media or online services.
 
+You should follow the [3-2-1 backup model](https://www.cisa.gov/sites/default/files/publications/data_backup_options.pdf) as outlined by CISA.
+
+[Time Machine](https://support.apple.com/104984) is the built-in tool for handling backups on macOS. Get an external drive or network drive to back up to and [encrypt](https://support.apple.com/guide/mac-help/keep-your-time-machine-backup-disk-secure-mh21241) your backups.
+
+You can also do backups manually if you want.
```

Comment:
This advice is not really actionable, remove it or justify.

Reply:
removed